docs/guides/postinstall.md

```diff
@@ -124,6 +124,26 @@ For other distros:
 
 - Compile [`tiny-dfr`](https://github.com/kekrby/tiny-dfr) yourself if your distro don't have that packaged yet.
 
+## Change default touchbar mode without tiny-dfr
+
+Run the following command, replacing `1` with the number corresponding to the desired touchbar mode.
+
+|Number|Default   |Fn      |
+|------|----------|--------|
+|0     |Function  |Ignore  |
+|1     |Media     |Function|
+|2     |Function  |Media   |
+|3     |Media     |Ignore  |
+|4     |Escape key|Ignore  |
+
+```bash
+cat <<EOF | sudo tee /etc/modprobe.d/tb.conf
+options hid-appletb-kbd mode=1
+EOF
+sudo modprobe -r hid-appletb-kbd
+sudo modprobe hid-appletb-kbd
+```
+
```

Comment:
add instructions for nixos while we're at it

```suggestion

??? info "NixOS"

    If you use NixOS, instead of running the command, add the following to your configuration. Modify `mode=1` to your desired value.
    ```nix
    {...}: {
      boot.extraModprobeConfig = ''
        options hid-appletb-kbd mode=1
      '';
    }
    ```
```